openfe/utils/system_probe.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import os
+import socket
+import subprocess
+
+import psutil
+
+
+def _get_disk_usage() -> dict[str, dict[str, str]]:
+
+    output = subprocess.check_output(["df", "-h"]).decode("utf-8")
```

Comment:
Forgive my naive question - if we 're already using psutil, is there a reason we can't just call `psutil.disk_usage` and divide things up as necessary?

Edit: or even `shutil`!

Reply:
This is a good question, but all of those require a path, and we don't necessarily know where these files will be written, but df will show disk usage at all mount points